src/mount_efs/__init__.py

```diff
@@ -1085,9 +1085,14 @@ def _stunnel_bin():
 
 def find_command_path(command, install_method):
     try:
-        env_path = (
-            "/sbin:/usr/sbin:/usr/local/sbin:/root/bin:/usr/local/bin:/usr/bin:/bin"
-        )
+        if check_if_platform_is_mac():
```

Comment:
Since most of the platform is not Mac, can you change the logic to

```python
if not check_if_platform_is_mac():
   ...
else:
   ...
```

Also can you add some comment on why the path is different, and what version of homebrew is that change introduced? e.g. you can link https://brew.sh/2021/02/05/homebrew-3.0.0/

Reply:
Ack. I'll reverse and add details in comments.